runtime/turing/src/lib.rs

```diff
@@ -1121,7 +1121,7 @@ impl_runtime_apis! {
 		fn fees(encoded_xt: Bytes) -> Result<Balance, Vec<u8>> {
 			let extrinsic: <Block as BlockT>::Extrinsic = Decode::decode(&mut &*encoded_xt).unwrap();
 			if let Call::AutomationTime(pallet_automation_time::Call::schedule_xcmp_task{
-				execution_times, para_id, currency_id, encoded_call_weight, ..
+				schedule, para_id, currency_id, encoded_call_weight, ..
```

Comment:
@chrisli30 API change number 3
The `xcmp_handler.fees` rpc expects encoded calls to include the `ScheduleParam` enum. This might be a breaking change in our js sdk, but I'm not sure who, if anyone, is consuming this.

Reply:
Yes, this is a breaking change. I will coordinate with mangata.